Source/Urho3D/Engine/Console.cpp

```diff
@@ -376,14 +376,10 @@ void Console::HandleTextFinished(StringHash eventType, VariantMap& eventData)
         // Send the command as an event for script subsystem
         using namespace ConsoleCommand;
 
-#if URHO3D_CXX11
-        SendEvent(E_CONSOLECOMMAND, P_COMMAND, line, P_ID, static_cast<Text*>(interpreters_->GetSelectedItem())->GetText());
-#else
         VariantMap& newEventData = GetEventDataMap();

```

Comment:
I was using my iPhone to review the PR. The small viewport and the mobile version has limit my ability to do the searches. Earlier I thought you just remove the guard and causing the event being sent twice. Only now I see you just remove the `then` block. My apology. Perhaps you could try search for "variadic" or "template initialization" or something like that in the issue tracker and may be also in the forum. Not really sure where the discussion took place anymore or with whom.